interfaces/seccomp/template.go

```diff
@@ -73,6 +73,7 @@ chown - u:root g:root
 chown32 - u:root g:root
 fchown - u:root g:root
 fchown32 - u:root g:root
+fchownat - - u:root g:root
```

Comment:
Do we need the trailing `-` for flags?

Reply:
This is not needed, a "-" has the same effect as omitting it.